saleor/graphql/shop/mutations.py

```diff
@@ -83,6 +89,27 @@ def mutate(cls, root, info, input):
         return ShopDomainUpdate(shop=Shop(), errors=errors)
 
 
+class ShopFetchTaxRates(BaseMutation):
+    shop = graphene.Field(Shop, description='Updated Shop')
+
+    class Meta:
+        description = 'Fetch tax rates'
+
+    @classmethod
+    @permission_required('site.manage_settings')
+    def mutate(cls, root, info):
+        errors = []
+        try:
+            call_command('get_vat_rates')
+        except ImproperlyConfigured as exc:
+            logger.exception(exc)
```

Comment:
IMHO, there is no need to log this since the error is explicitly returned to the API by ```errors=errors```

Reply:
`logger.exception` accepts a log message as its argument, you don't need to pass the actual exception.